### PR TITLE
Add Creating Gibwork Task

### PR DESCRIPTION
**Pull request by DogukanGun**
The feature to create tasks on Gibwork has been added.. 
 This pull request was created for https://app.gib.work/bounties/6963c538-dc0b-4a3c-9cd5-0a18c0b20d9e in an attempt to solve a bounty #48 . Payment for the bounty is immediately sent to the contributor after merge.

### CODE REVIEW

**Comment by thearyanag** [comment]
where is the agent signing the transaction?

**Comment by DogukanGun** [comment]
> where is the agent signing the transaction?

Hi @thearyanag , the tool operates by making requests to a REST API. Therefore, it does not involve signing transactions.

File: src/tools/create_gibwork_task.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            })
+        });
+
+        const responseData = await apiResponse.json();
+
+        return {
+            success: true,
+            taskId: responseData.taskId,
+            serializedTransaction: responseData.serializedTransaction
+        };
```
Comment (thearyanag) [inline src/tools/create_gibwork_task.ts:50]: @DogukanGun then why is here a serializedTransaction returned? the user needs to transfer the tokens to gibwork contracts, right?

Comment (DogukanGun) [reply src/tools/create_gibwork_task.ts:69]: @thearyanag  Yes, you're absolutely right, I missed that. My apologies!

**Pull request closed without merging**